lessons/Early-Math/exponent-expressions-with-negatives/langs/en-us/index.adoc

```diff
@@ -0,0 +1,100 @@
+= Evaluating Exponent Expressions with Negative Values
+
+@description{Students use Circles of Evaluation to analyze the structure of expressions with exponents and negative values.}
+
+@lesson-prereqs{evaluating-exponents}
+
+@keywords{Circle of Evaluation, }
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Determine the order in which we negate and apply exponents in expressions both with and without grouping symbols.
+* Evaluate expressions involving exponents and negative values.
+* Evaluate exponent expressions with negatives and variables.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyse the structure of expressions with exponents and negative values.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+
+== Exponent Expressions with Negatives @duration{30 minutes}
+
+=== Overview
+
+Students apply Circles of Evaluation to analyze expressions with exponents and negatives.
+
+=== Launch
+
+[.lesson-instruction]
+--
+- Look closely at each of these expressions, considering the effect of the negation and the exponent on the outcome: @math{-3^2}, @math{(-3)^2}, and @math{-(3^2)}
+- Are any of these three expressions equivalent? How do you know? Draw Circles to help you.
+** _We know that in an expression with no grouping symbols, we evaluate the exponent first - before the negation. Therefore, the first and third expression are equivalent; they both evaluate to @math{-9}. The second expression evaluates to @math{9}. See Circles below._
+[.embedded, cols="^.^1,^.^1", grid="none", stripes="none", frame="none"]
+|===
+|@math{-3^2} or @math{-(3^2)}				| @math{(-3)^2}
+|@show{(coe  '(negate (num-expt 3 2)))}		| @show{(coe  '(num-expt (negate 3) 2) )}
+|===
```

Comment:
I'm not sure how the circles would help students to know that -3^2 and -(3^2)... but I do think the visual draws a powerful distinction from (-3)^2. That makes me think we want every student to see these circles, so perhaps they belong on a workbook page.

Wondering if you might instead want to show coe for -(3^2) and (-3)^2, have them describe how they're different. Then tell them that -3^2 has the same circle as one of the two and have them try to decide which and justify their thinking.